src/GUI/EFCorePowerTools/Handlers/ReverseEngineer/TableListBuilder.cs

```diff
@@ -39,21 +39,6 @@ public List<TableInformationModel> GetTableDefinitions(bool useEFCore5)
                 tables = launcher.GetTables(_connectionString, _databaseType, _schemas);
             }
 
-            foreach (var item in tables)
-            {
-                if (item.IsProcedure)
-                {
-                    item.HasPrimaryKey = true;
-                    continue;
-                }
-
-                if (!item.HasPrimaryKey)
-                {
-                    item.HasPrimaryKey = true;
-                    item.ShowKeylessWarning = true;
```

Comment:
Are views displayed with the icon and warning now?

Reply:
Yes, those fix were needed by the TableInformationViewModel so I moved them there